examples/core/Graphormer/README.md

```diff
@@ -0,0 +1,28 @@
+Graphormer
+==============================
+
+* paper link: [https://arxiv.org/abs/2106.05234](https://arxiv.org/abs/2106.05234)
+
+## Requirements
+- accelerate
+- transformers
+- ogb
+
+## Dataset
+
+Task: Graph Property Prediction
+
+|   Dataset   | #Graphs | #Node Feats | #Edge Feats | Metric |
+| :---------: | :-----: | :---------: | :---------: | :-----: |
+| ogbg-molhiv | 41,127 |      9      |      3      | ROC-AUC |
+
+How to run
+----------
+
+```bash
+accelerate launch --multi_gpu --mixed_precision=fp16 train.py
+```
```

Comment:
Add a note (the script automatically download & load pre-trained weights on PCQM4Mv2) below?